torch/_torch_docs.py

```diff
@@ -745,6 +745,23 @@ def merge_dicts(*dicts):
     tensor([0.0000, 0.0000, 0.0000, 1.0000, 1.0000, 0.0000, 0.5000])
 """)
 
+add_docstr(torch.bitwise_not,
```

Comment:
Needs entry in https://github.com/pytorch/pytorch/blob/master/docs/source/torch.rst